app/services/data/get-reductions.js

```diff
@@ -0,0 +1,25 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+
+module.exports = function (id) {
+  return knex('reductions')
+    .join('reduction_reason', 'reductions.reduction_reason_id', 'reduction_reason.id')
+    .join('reduction_category', 'reduction_reason.category_id', 'reduction_category.id')
+    .modify(function (whereBlock) {
```

Comment:
Could move this where block to above the query to make it a bit clearer?  Like we have in the overview queries.

Reply:
Done.